app/models/concerns/sync_loggable.rb

```diff
@@ -0,0 +1,18 @@
+module SyncLoggable
+  extend ActiveSupport::Concern
+
+  def latest_sync_log(simple_id, simple_model)
+    SyncLog.where(simple_id: simple_id, simple_model: simple_model).order(synced_at: :desc).first
+  end
+
+  def sync_status(sync_log)
+    return :unsynced unless sync_log.present?
```

Comment:
@govindkrjoshi What do you think of using `if sync_log.blank?` over `unless sync_log.present?`